generator/testdata/rust/protobuf/golden/iam/v1/src/transport.rs

```diff
@@ -47,8 +47,12 @@ impl crate::traits::IAMPolicy for IAMPolicy {
     ) -> Result<crate::model::Policy> {
         let options = options.set_default_idempotency(reqwest::Method::POST.is_idempotent());
         let builder = self.inner.builder(
-            reqwest::Method::POST, format!("/v1/{}:setIamPolicy"
-               , req.resource
+            reqwest::Method::POST,
+            format!("/v1/{}:setIamPolicy"
+            ,
+                req
+                            .resource
```

Comment:
I couldn't find a way to skip white spaces in the template. Any suggestion? 
At least this only affects the test data, `cargo fmt` takes care of this for the generated code.

Reply:
See above.